### DISCUSSION TITLE
Zooming Map

### DISCUSSION THREAD

**Discussion by Marty56** [mytourbook/mytourbook#1275] [General]
Hi,

I am facing two bugs when trying to zoom in the 2D Map.
1. When zoom into the map (on MacOS), the map stays not centered. It is jumping to a totally different center. 
Is there any option how the correct zooming behaviour?
2. Another issue is zooming by mouse movement. This method has extrem zoom steps and is practically therefore not usable especially in conjunction with the bug explained above.

Help would be appreciated.

Update: I have discovered the button switching between various zoom options.

**Comment by FJBDev** [comment]
>When zoom into the map (on MacOS), the map stays not centered. It is jumping to a totally different center.
Is there any option how the correct zooming behaviour?

Which zooming method do you have selected in the 2D map ?
Below is what I have, as the tooltip says, when zooming it will use the mouse position as the center
![image](https://user-images.githubusercontent.com/25235665/217678383-7c943d5b-a2f5-4f65-826f-49fc03446745.png)

**Comment by Marty56** [comment]
Thanks.
